examples/pubsub/README.md

```diff
@@ -19,12 +19,12 @@ docker run -d --rm --hostname my-rabbitmq --name my-rabbitmq \
     rabbitmq:3-management
 
 # Run the example
-npm run start:dapr-http
+dapr run --app-protocol http
```

Comment:
In this PR, we have moved from npm scripts to running dapr directly, but looks like we are not setting `app-id`, `app-port`, and only running a side-car. How is the actual application being invoked? Earlier, the npm script was doing this:
```bash
dapr run --app-id example-invocation --app-port 50051 --app-protocol grpc npm run start
```

Reply:
@shubham1172 @XavierGeerinck we have the following in configuration/package.json... should I copy it to the readme as well??

 ```
  "start:dapr-grpc": "dapr run --app-id example-config --app-port 50051 --app-protocol grpc npm run start",
  "start:dapr-http": "dapr run --app-id example-config --app-port 50051 --app-protocol http npm run start"
```

Reply:
We can instead use the npm script directly everywhere. 
```
npm run start:dapr-http
```
And we can mention that for grpc, use
```
npm run start:dapr-grpc
```
What do you think?

Reply:
Agreed with @shubham1172 but maybe we should add a footnote on what the command executes? Ensuring people learn more about what `dapr run` is and how it works?

Reply:
Yes, that will be nice! It can even be in this format.
```bash
# directly using dapr run
dapr run --app-id example-config --app-port 50051 --app-protocol grpc npm run start
# or, using npm script
npm run start:dapr-http
```

Reply:
Loving this! but I think you mean `dapr run --app-id example-config --app-port 50051 --app-protocol http npm run start`  then since it's `start:dapr-http` right?

Reply:
Oh yes, I copied the wrong command. Absolutely!